deluge/ui/console/modes/preferences/preference_panes.py

```diff
@@ -241,9 +245,18 @@ def create_pane(self, core_conf, console_config):
         self.outto.set_depend(outrand, inverse=True)
 
         self.add_header(_('Interface'), space_above=True)
```

Comment:
should this really be listen interface maybe? since "interface" setting is for the daemon interface to listen on.

Reply:
The daemon setting is rarely something a user will specify. However, if you have added Outgoing Interface then yes changing this to Incoming Interface .

Reply:
This would also make the heading match gtkui and webui. So it will all look more uniform.